api/doc/action-audit-queries.md

```diff
@@ -0,0 +1,212 @@
+
+### Agent Types
+There are three agent types currently:
+- USER: standard AoU users. 
+- ADMINISTRATOR: Users with Admin privileges. Used for BYPASS events and the like
+- SYSTEM: Some part of the All of Us software stack. Typically used for automated events,
+or events that happen as a consequence of some other event.
+
+### Target Types
+A Target is a thing being operated on during an Action. The following target types are largely
+self-explanatory. The design document has additional details.
+ - USER
+ - WORKBENCH
+ - PROFILE
+ - ACCOUNT
+ - NOTEBOOK
+ - NOTEBOOK_SERVER
+ - DATASET
+ - CONCEPT_SET
+ - COHORT
+ - CREDIT
+ - WORKSPACE
+
+## Target Properties
+The target properties are key-value string pairs and are not in a strictly
+controlled schema, but are rather enforced by application constraints. Each target type 
+supports one or more properties, and these are outlined below:
+
+| Target Type | Property  |
+|-------------|----------|
+| WORKSPACE   | name     |
+| WORKSPACE | intended_study |
+| WORKSPACE | creator | 
+| WORKSPACE | additional_notes | 
+| WORKSPACE | anticipated_findings | 
+| WORKSPACE | disease_of_focus | 
+| WORKSPACE | reason_for_all_of_us | 
+| WORKSPACE | namespace | 
+| WORKSPACE | cdr_version_id | 
+| PROFILE | user_name |
+| PROFILE | contact_email |
+| PROFILE | data_access_level |
+| PROFILE | given_name |
+| PROFILE | phone_number |
+| PROFILE | current_position |
+| PROFILE | organization |
+| PROFILE | disabled |
+| PROFILE | about_you |
+| PROFILE | area_of_research |
+| PROFILE | institutional_affiliations |
+| PROFILE | demographic_survey |
+| PROFILE | address |
+| ACL | ACCESS_LEVEL |
+
+## Example Queries
+These queries are the same for every environment wiht the exception of the BigQuery
+Dataset and Table names. The fully qualified table name is of the form
+`$GCP_PROJECT.$BIGQUERY_DATASET.$TABLE`. You'll need appropriate permissions on the
+project, dataset, and BigQuery service for these to work.
+
+| Environment | Fully Qualified Table Name |
+|---|----|
+| Test | `all-of-us-workbench-test.workbench_action_audit_test.workbench_action_audit_test` |
+| Prod | `all-of-us-rw-prod.workbench_action_audit_prod.workbench_action_audit_prod` |
+
+### All Events by Time`
+All events by anyone, ordered by time ascending.
+```genericsql
+SELECT
+  TIMESTAMP_MILLIS(CAST(jsonPayload.timestamp AS INT64)) as event_time,
+  jsonPayload.agent_type AS agent_type,
+  CAST(jsonPayload.agent_id AS INT64) AS agent_id,
+  jsonPayload.agent_email AS agent_email,
+  jsonPayload.action_id AS action_id,
+  jsonPayload.action_type AS action_type,
+  jsonPayload.target_type AS target_type,
+  CAST(jsonPayload.target_id AS INT64) AS target_id,
+  jsonPayload.target_property AS target_property,
+  jsonPayload.prev_value AS prev_value,
+  jsonPayload.new_value AS new_value
+FROM `all-of-us-rw-prod.workbench_action_audit_prod.workbench_action_audit_prod`
+ORDER BY event_time, agent_id
```

Comment:
maybe add action_type to order by for cases like `DUPLICATE_FROM ` and `DUPLICATE_TO` which show unordered for same event_time and action_id?
```suggestion
ORDER BY event_time, agent_id, action_type
```